website_ore/controllers/main.py

```diff
@@ -1153,6 +1200,10 @@ def get_membre_information(self, membre_id, **kw):
                 "actual_month_bank_hours": membre_id.bank_month_time,
                 "is_favorite": is_favorite,
                 "introduction": membre_id.introduction,
+                "description": membre_id.description,
+                "motivation_membre": membre_id.motivation_membre,
+                "interet": [rec.name for rec in membre_id.interet],
+                "langue": [rec.name for rec in membre_id.langue_parle],
```

Comment:
Envoyer la liste des ID au client
[{"name": rec.name, "id":rec.id for rec} in membre_id.langue_parle]